### PR TITLE
Add App Runner definitions

### PR DESCRIPTION
**Pull request by hunterwerlla**
See: https://github.com/aws/aws-toolkit-jetbrains/blob/36c3456e51e695cb6e38f993ca02d498b5cdfe9b/jetbrains-core/resources/telemetryOverride.json

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: telemetry/definitions/commonDefinitions.json [hunk 1]
```diff
@@ -250,6 +256,41 @@
...
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "apprunner_startDeployment",
+            "description": "Start a new deployment for an AppRunner service",
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "apprunner_viewApplicationLogs",
+            "description": "View the AppRunner service logs"
```
Comment (JadenSimon) [inline telemetry/definitions/commonDefinitions.json:288]:
```suggestion
            "description": "View the AppRunner application logs"
```

**Review by JadenSimon — APPROVED** [review]

File: telemetry/definitions/commonDefinitions.json (hunk 1 above)
Comment (hunterwerlla) [reply telemetry/definitions/commonDefinitions.json:288]: The top level construct is called `service`

Comment (JadenSimon) [reply telemetry/definitions/commonDefinitions.json:288]: I know I thought App Runner had logs for the running application plus logs for the service resource.

Comment (hunterwerlla) [reply telemetry/definitions/commonDefinitions.json]: well this is confusing https://docs.aws.amazon.com/apprunner/latest/dg/monitor-cwl.html

service = system logs
application = logs for a service

So changed to match the service naming scheme and updated the descriptions

**Review by JadenSimon — APPROVED** [review]

**Pull request merged**